frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -23,6 +23,10 @@ export const getLabels = <A extends K8sResourceCommon = K8sResourceCommon>(
   value: A,
   defaultValue?: K8sResourceCommon['metadata']['labels'],
 ) => (_.has(value, 'metadata.labels') ? value.metadata.labels : defaultValue);
+export const getLabel = <A extends K8sResourceCommon = K8sResourceCommon>(
+  value: A,
+  label: string,
+) => getLabels(value, {})[label];
```

Comment:
this function is for the whole code base so we should rather not create a new object on each call